### ISSUE TITLE
Series in result

### ISSUE THREAD

**Issue by kdsystem** [hatamiarash7/Mikrotik-Exporter#8]
Hi.
It seems that dashboard couldn't display more than one dhcp pool and more than one wifi ssid statistics.
![image](https://user-images.githubusercontent.com/11457765/92222987-7ac70080-eea8-11ea-9bde-d97547d448a9.png)
Please, what should i do?

**Comment by hatamiarash7** [comment]
Yes that's a simple dashboard. You should edit panels for this purpose.  
Let me check tomorrow.

**Comment by kdsystem** [comment]
Yes, thank you - it would be great.
